### PR TITLE
Feedback for TJ's Killer Article

### PR DESCRIPTION
**Pull request by alyssamichelle**
Wonderful comparison article. Seriously, I loved it. 

I think it's funny you like how Angular handles styles. It's actually my biggest issue with Angular. I use `ViewEncapsulation.None` often. I feel like the way that Angular scopes styles per component breaks down the whole "cascading" part of CSS. I anticipate my styles being global and having different weights. So it throws me when I forget to disable view encapsulation. lol

I wanted to comment on the flow of your article. Your transitions from one section to the next are seamless. You really are a good writer, TJ. I'm sorry about submitting all the changes like this. After making this, I realized comments might be easier for you to work with. Merp :/

### CODE REVIEW

File: kendo/angular-v-react/article.md [hunk 1]
```diff
@@ -44,13 +42,14 @@ src
 │   ├── app.component.ts
 │   ├── app.module.ts
 │   └── helloworld
-│       ├── helloworld-routing.component.ts
```
Comment (alyssamichelle) [inline kendo/angular-v-react/article.md:47]: I don't think there would be a routing file here for a hello world app.

File: kendo/angular-v-react/article.md [hunk 2]
```diff
@@ -449,7 +448,7 @@ It’s also worth establishing conventions within your own organization so your
 
 If you came to this article looking for me to strongly recommend one framework over another, well then I have some bad news for you. Despite the differences between Angular and React, my single biggest finding is that, at the end of the day, the two frameworks are remarkably similar.
 
-React and Angular both do data binding; they both have a rich templating syntax; they both handle events; they both have routers; and they both let you use modern JavaScript features.
```
Comment (alyssamichelle) [inline kendo/angular-v-react/article.md:452]: Can you use semicolons like this?

File: kendo/angular-v-react/article.md (hunk 1 above)
Comment (tjvantoll) [reply kendo/angular-v-react/article.md:47]: Good point. I’ll note that this is a hello world app with a routing structure in place, or something like that.

File: kendo/angular-v-react/article.md (hunk 2 above)
Comment (tjvantoll) [reply kendo/angular-v-react/article.md:452]: I _think_ so? But the commas read cleaner so I like the change.

**Pull request closed without merging**